README.md

```diff
@@ -110,8 +110,14 @@ so it runs anywhere Git and Bash runs.
 * __Autocompletion for [Fish shell](http://fishshell.com)__
 
     ```
-    mkdir -p ~/.config/fish/completions/
-    curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
+    $ git town install-fish-autocompletion
+    ```
+
+    or manually
+
+    ```
+    $ mkdir -p ~/.config/fish/completions/
+    $ curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
```

Comment:
It already lives in the `git-town` man page and markdown page. By updating the summary of git-town as I suggested (in the README and on the man page and markdown page) I believe that points users to it.